src/Template/Element/advanced_search_form.ctp

```diff
@@ -357,6 +357,10 @@ echo $this->Form->create('AdvancedSearch', [
                    href="http://en.wiki.tatoeba.org/articles/show/text-search">
             <?= __('More search options') ?>
         </md-button>
+
+        <md-button type="submit" class="md-primary" formaction="/eng/sentences/advanced_search">
```

Comment:
Whoops, just realized this is hard-linked to English.

Reply:
That’s right. You should be able to create the correct link using the [URL helper](https://book.cakephp.org/3/en/views/helpers/url.html#generating-urls).